unit_tests/fields/field_algo_base_test.cpp

```diff
@@ -581,6 +582,9 @@ TEST(Field, init_from_input) {
 	::testing::FLAGS_gtest_death_test_style = "threadsafe";
 	Profiler::initialize();
 
+    // necessary only for registration of FieldFormula to Abstract
+    FieldFormula<3, FieldValue<3>::TensorFixed > formula_field;
```

Comment:
Do not understand why these would be necessary? Registration should happen due to explicit instantiation.